tests/features/project.feature

```diff
@@ -40,10 +23,35 @@ Feature: Project content creation
     And I fill in "Subject" with "EU financing"
     And I fill in "Author" with "European Patent Office"
     And I fill in "Body text" with "Body text"
-    And I fill in "Coordinators" with "Coordinators stakeholder"
-    And I fill in "Participants" with "Participants stakeholder"
     And I fill in "Teaser" with "Project teaser text" in the "Alternative titles and teaser" region
     And I fill in "Content owner" with "Committee on Agriculture and Rural Development"
+
+    # Fill in Stakeholder fields by inline entity form for Coordinators field.
+    And I fill in "Name" with "Coordinators stakeholder" in the "Project coordinators" region
+    And I fill in "Acronym" with "Acronym of the Coordinator" in the "Project coordinators" region
+    And I fill in "Use existing media" with "Image 1" in the "Project coordinators" region
+    And I select "Belgium" from "Country" in the "Project coordinators" region
+    And I wait for AJAX to finish
+    And I fill in "Company" with "Ma société 1" in the "Project coordinators" region
```

Comment:
Can French language be used in the test content?

Reply:
These are the types of values that will be used, so its nice to use them in the tests

Reply:
I'd use english on an english translation and french on a french one, let's stick to english unless we are testing a translation.